azurepipelines/build/docker/adu-debian-amd64-build.yml

```diff
@@ -52,30 +53,58 @@ pr:
             - LICENSE.md
             - .clang-format
             - .cmake-format.json
-            - tools/*
             - docker/*
-            - scripts/*
             - licenses/*
 
-jobs:
-    - job: BuildAduAgent_debian10
-      displayName: "Build ADU Agent - Debian 10 AMD64"
-      timeoutInMinutes: 360
-      cancelTimeoutInMinutes: 360
-      pool: aduc_1es_client_pool
-      steps:
-          - template: ../templates/adu-docker-build-steps.yml
-            parameters:
-                targetOs: "debian10"
-                targetArch: "amd64"
+resources:
+  repositories:
+  - repository: 1ESPipelineTemplates
```

Comment:
anything comment worthy here by adding the 1ES templates and extending them?

Reply:
Added to the work of yaml cleanup